src/socket/IOProxy.ts

```diff
@@ -111,7 +111,8 @@ export default class IOProxy {
     await this.getStorage(peerId).subscribeForAllConversations(conversations)
   }
 
-  public registerUserCertificate = async (serviceAddress: string, userCsr: string) => {
+  public registerUserCertificate = async (serviceAddress: string, userCsr: string, id: string) => {
```

Comment:
What is this 'id'? Is this communityId? If so, can you rename it (as it is in other methods)?